src/platform-apps/channels/channel-view-container.test.tsx

```diff
@@ -98,6 +99,26 @@ describe('ChannelViewContainer', () => {
     });
   });
 
+  it('should call sendMessage when textearea is clicked', () => {
+    const sendMessage = jest.fn();
+    const message = 'test message';
+    const mentionedUser = 'ef698a51-1cea-42f8-a078-c0f96ed03c9e';
+
+    const wrapper = subject({
+      sendMessage,
+      channelId: 'the-channel-id',
+      channel: { hasMore: true, name: 'first channel' },
+    });
+
+    wrapper.find(ChannelView).first().prop('sendMessage')(message);
+
+    expect(sendMessage).toHaveBeenLastCalledWith({
```

Comment:
i think that `toHaveBeenLastCalledWith` might be the wrong expectation here. i think in this case we want to make sure that it was called once and only once.